src/utils/timePassedFormating.ts

```diff
@@ -0,0 +1,11 @@
+import moment from 'moment';
```

Comment:
Moment is no longer maintained. See: https://momentjs.com/docs/#/-project-status/